broken bootstrap build

We have an error:

```
/home/runner/work/gccrs/gccrs/gccrs-build/./prev-gcc/xg++ -B/home/runner/work/gccrs/gccrs/gccrs-build/./prev-gcc/ -B/usr/local/x86_64-pc-linux-gnu/bin/ -nostdinc++ -B/home/runner/work/gccrs/gccrs/gccrs-build/prev-x86_64-pc-linux-gnu/libstdc++-v3/src/.libs -B/home/runner/work/gccrs/gccrs/gccrs-build/prev-x86_64-pc-linux-gnu/libstdc++-v3/libsupc++/.libs  -I/home/runner/work/gccrs/gccrs/gccrs-build/prev-x86_64-pc-linux-gnu/libstdc++-v3/include/x86_64-pc-linux-gnu  -I/home/runner/work/gccrs/gccrs/gccrs-build/prev-x86_64-pc-linux-gnu/libstdc++-v3/include  -I/home/runner/work/gccrs/gccrs/libstdc++-v3/libsupc++ -L/home/runner/work/gccrs/gccrs/gccrs-build/prev-x86_64-pc-linux-gnu/libstdc++-v3/src/.libs -L/home/runner/work/gccrs/gccrs/gccrs-build/prev-x86_64-pc-linux-gnu/libstdc++-v3/libsupc++/.libs  -fno-PIE -c  -DIN_GCC_FRONTEND -g -O2 -fno-checking -gtoggle -DIN_GCC     -fno-exceptions -fno-rtti -fasynchronous-unwind-tables -W -Wall -Wno-narrowing -Wwrite-strings -Wcast-qual -Wmissing-format-attribute -Woverloaded-virtual -pedantic -Wno-long-long -Wno-variadic-macros -Wno-overlength-strings -Werror -fno-common  -DHAVE_CONFIG_H -I. -Irust -I../../gcc -I../../gcc/rust -I../../gcc/../include -I../../gcc/../libcpp/include -I../../gcc/../libcody  -I../../gcc/../libdecnumber -I../../gcc/../libdecnumber/bid -I../libdecnumber -I../../gcc/../libbacktrace   -o rust/rust-constexpr.o -MT rust/rust-constexpr.o -MMD -MP -MF rust/.deps/rust-constexpr.TPo -std=c++11 -Wno-unused-parameter -Werror=overloaded-virtual -I ../../gcc/rust -I ../../gcc/rust/lex -I ../../gcc/rust/parse -I ../../gcc/rust/ast -I ../../gcc/rust/analysis -I ../../gcc/rust/backend -I ../../gcc/rust/expand -I ../../gcc/rust/hir/tree -I ../../gcc/rust/hir -I ../../gcc/rust/resolve -I ../../gcc/rust/util -I ../../gcc/rust/typecheck -I ../../gcc/rust/checks/lints -I ../../gcc/rust/checks/errors -I ../../gcc/rust/checks/errors/privacy -I ../../gcc/rust/util -I ../../gcc/rust/metadata ../../gcc/rust/backend/rust-constexpr.cc
../../gcc/rust/backend/rust-constexpr.cc: In function ‘tree_node* Rust::Compile::eval_array_reference(const constexpr_ctx*, tree, bool, bool*, bool*)’:
../../gcc/rust/backend/rust-constexpr.cc:1675:53: error: spurious trailing punctuation sequence ‘.’ in format [-Werror=format-diag]
 1675 |   sorry ("array size expression is not supported yet.");
      |                                                     ^
```

--- a/gcc/rust/backend/rust-constexpr.cc
+++ b/gcc/rust/backend/rust-constexpr.cc
@@ -1672,7 +1672,7 @@ eval_array_reference (const constexpr_ctx *ctx, tree t, bool lval,
      initializer, it's initialized from {}.  But use build_value_init
      directly for non-aggregates to avoid creating a garbage CONSTRUCTOR.  */
   tree val = NULL_TREE;
-  sorry ("array size expression is not supported yet.");
+  sorry ("array size expression is not supported yet");
 
   constexpr_ctx new_ctx;
   if (is_really_empty_class (elem_type, /*ignore_vptr*/ false))

